core/metadata/metadata.go

```diff
@@ -1,55 +1,44 @@
-/*******************************************************************************
- * Copyright 2017 Dell Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License
- * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- *
- * @microservice: core-metadata-go service
- * @author: Spencer Bull & Ryan Comer, Dell
- * @version: 0.5.0
- *******************************************************************************/
-package main
+//
+// Copyright (c) 2018
+// Cavium
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+package metadata
 
 import (
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
-	"os"
 	"strconv"
 	"strings"
 	"time"
 
-	consulclient "github.com/edgexfoundry/edgex-go/support/consul-client"
 	enums "github.com/edgexfoundry/edgex-go/core/domain/enums"
+	consulclient "github.com/edgexfoundry/edgex-go/support/consul-client"
 	logger "github.com/edgexfoundry/edgex-go/support/logging-client"
 	notifications "github.com/edgexfoundry/edgex-go/support/notifications-client"
 )
 
 // DS : DataStore to retrieve data from database.
 var DS DataStore
-var notificationsClient = notifications.NotificationsClient{}
 var loggingClient logger.LoggingClient
+var notificationsClient = notifications.NotificationsClient{}
 
-func main() {
-	start := time.Now()
-
-	// Load configuration data
-	err := readConfigurationFile(CONFIG)
-	if err != nil {
-		fmt.Printf("Could not read configuration file(%s): %#v\n", CONFIG, err)
-		os.Exit(1)
+// Heartbeat for the metadata microservice - send a message to logging service
+func heartbeat() {
+	// Loop forever
+	for true {
```

Comment:
see comment above about Go idiomatic infinite loops.

Reply:
Drasko - let us just get the code moved over from the multiple repos before we start trying to clean up the world under one PR!  We agreed we just wanted to get mono repo up with existing code.  Let's get everything working before we burden the PRs with more requests.